packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -11,7 +11,7 @@ $detailsList-item-selected-hover-color: $listItemSelectedHoverColor;
 .root {
   @include focus-border();
 
-  display: inline-block;
+  display: flex;
```

Comment:
ahh yup :) I "had it working" but didn't realize my checkmark was in the wrong place.